generator/AWSPSGeneratorLib/HelpMaterials/Examples/IAM/Add-IAMUserTag.xml

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>$tag = New-Object -TypeName Amazon.IdentityManagement.Model.Tag
+$tag.Key = 'abac'
+$tag.Value = 'testing'
+Add-IAMUserTag -UserName joe -Tag $tag</code>
```

Comment:
Considering how simple this object is, I would probably simply do `-Tag @{ Key = 'abac'; Value = 'testing'}`